ovirt-engine-dwh.spec.in

```diff
@@ -17,39 +17,15 @@
 %global jackson_core jackson-core
 %global jackson_databind jackson-databind
 %global jackson_annotations jackson-annotations
-%if 0%{?fedora}
-%global ovirt_install_systemd 1
-%global ovirt_install_systemv 0
-%global commons_collections apache-commons-collections
-%global dom4j dom4j
-%endif
 
 %if 0%{?rhel}
-    %if 0%{?rhel} >= 8
-        %global ovirt_install_systemd 1
-        %global ovirt_install_systemv 0
-        %global commons_collections apache-commons-collections
-        %if 0%{?centos}
-            %global dom4j ovirt-engine-wildfly
-        %else
-            %global dom4j eap7-dom4j
-        %endif
+    %global ovirt_install_systemd 1
```

Comment:
You can move `%if 0%{?rhel}` down keeping everything common so it will be picked up by Fedora as well, the only real difference is between RHV and oVirt where oVirt uses wildfly and RHV uses eap.
Perhaps do as in ovirt-engine: https://github.com/oVirt/ovirt-engine/blob/master/ovirt-engine.spec.in#L45

Reply:
> Perhaps do as in ovirt-engine: https://github.com/oVirt/ovirt-engine/blob/master/ovirt-engine.spec.in#L45

You mean define rhv_build parameter ?

Reply:
yes

Reply:
Thanks Sandro,
After consulting with Martin it was decided to leave as is.